framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -601,26 +597,13 @@ protected void addInformationFromPreconditions(
      *
      * @param annoFromContract a contract annotation that was written on a method declaration
```

Comment:
It is a contract annotation, but it wasn't necessarily written explicitly.  For example, it might be `@NonNull` when the user wrote `@EnsuresNonNull`.  (This comment also applies to other occurrences of this Javadoc.)